contributors/design-proposals/storage/volume-topology-scheduling.md

```diff
@@ -415,40 +436,31 @@ Kubernetes.
 With all this complexity, the library approach is the most feasible in a single
 release time frame, and aligns better with the current Kubernetes architecture.
 
-#### Downsides/Problems
-
-##### Expanding Scheduler Scope
```

Comment:
The "scope" here was referring to scheduler having to also handle scenarios where the user manually sets Pod.Spec.NodeName to bypass the scheduler.  This was because the original plan was to have this behavior change for all volume binding.

But we changed the API so that this new behavior can be controller explicitly through StorageClass.  And because of that, we decided that we don't need to support this use case of setting Pod.Spec.NodeName.  It was only considered previously for backwards compatibility.

Reply:
I was referring to the "expanding scheduler scope" from being just a pod scheduler to being a pod-and-volume scheduler. The downsides seem to be overlap with PV binder responsibilities, and requiring custom schedulers to also be pod-and-volume schedulers.